onnxscript/rewriter/ort_fusions/fuse_qkv.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Nit: suggest expanding the file name .. may something like "fuse_packed_qkv_gqa"